Sources/SwiftGenKit/Parsers/Fonts/FontsParser.swift

```diff
@@ -9,6 +9,14 @@ import Foundation
 import PathKit
 
 public enum Fonts {
+  public enum Option {
+    static let codePoints = ParserOption(
```

Comment:
Maybe name the option `glyphNames` instead of `codePoints`?

Reply:
Also, this parser option needs to be documented in `Documentation/Parsers/fonts.md` in the "Supported Options" section (you might need to merge develop into your branch first, as we renamed that documentation folder in a recent PR that was open and merged after you opened yours)